code-challenges.js

```diff
@@ -4,15 +4,45 @@
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+//create a function
+const fibonacci = () => {
+   // create an array that will hold the numbers
+   let sequence = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+   
+    // return that array
+   return sequence
+   
+}
+
+console.log(fibonacci())
+
 
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 9, 7, 199]
+// Expected output: [-9, 7, 9, 199]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+// create a function that takes an array as an argument
+//iterate throught the array and filter out any non numbers 
+// sort the array from least to greatest
+
+const oddNumberSorted = (arry) => {
+    let numberONly= arry.filter(value => {
+        return "number" === typeof value
+    })
+
+    let oddNumbers = numberONly.filter( value => value % 2 !== 0)
+
+    return oddNumbers.sort((a,b) => a-b)
+    
+}
+ 
```

Comment:
fantastic! this could be boiled down to one line but I like that your process is laid out here.